### ISSUE TITLE
Preserve branch arrow on currently selected line in disassembly

### ISSUE THREAD

**Issue by deklesen** [radareorg/cutter#507]
The branch arrows indicate the jump destinations. Most of the time, you follow the arrow from a specific instruction to see where it goes. If you have to scroll to follow it, the arrow disappears. It would be more useful if the arrow from the selected line (and maybe even the ones pointing to that line) would be preserved when scrolling. x64dbg does this, for example.

Related feature request when debugger is ready: color the arrow from selected line red, if the jump condition evaluates to true.
Thanks for all the hard work and have a good day!

**Comment by thestr4ng3r** [comment]
Not an easy issue imho. Should be solved when we have a new Disassembly Widget and arrows aren't drawn as text anymore.

**Comment by xarkes** [comment]
Oh sorry, I thought it was just to set the current block jump lines dashed so it's easier to see.

**Comment by ITAYC0HEN** [comment]
this is actually a bug. Arrows should always stay

**Comment by ITAYC0HEN** [comment]
@xarkes please see this. For the future refactoring of Disassembly widget

**Comment by ITAYC0HEN** [comment]
@xarkes please see this. For the future refactoring of Disassembly widget

**Comment by ITAYC0HEN** [comment]
@xarkes did you respect this in your solution?

**Comment by xarkes** [comment]
No it doesn't because currently nothing is cached. Once cache is supported for the disassembly widget, this would be easy to implement.

**Comment by ITAYC0HEN** [comment]
@xarkes @karliss @optizone I'm not sure but - is it already implemented after the disassembly refactoring? Or only partially

**Comment by xarkes** [comment]
Nope, the disassembly logic is still the same - only the offsets you see are disassembled. Nothing more nothing less.
